inspire/modules/workflows/tasks/submission.py

```diff
@@ -287,3 +288,58 @@ def add_files_to_task_results(obj, eng):
         obj.add_task_result(file_obj.name,
                             fileinfo,
                             "workflows/results/files.html")
+
+
+def create_curation_ticket(template, queue="Test"):
```

Comment:
I see a lot of similarities between this function and the  `create_ticket()` function. Is there any way we can re-use `create_ticket()` function or make it generic enough in an attempt to avoid code duplication?